src/workerd/api/node/tests/crypto_sign-test.js

```diff
@@ -0,0 +1,146 @@
+import {
+  createSign,
+  createVerify,
+  createPrivateKey,
+  createPublicKey,
+  sign,
+  verify,
+} from 'node:crypto';
+
+import { strictEqual, throws } from 'node:assert';
+
+const rsaSig =
+  '26bb4d9641ecec048b791322c6427f62f3f4e21f7198e9e7544c8a56af40' +
+  '27bcfe1b306291188f97ced0e3ceaa5ded1ae1406ec30e46a18434e55dc6' +
+  'c9f237e26b124bf7ec77e54483d7782b805aa9a74bbe0f4aa8658d7620e4' +
+  'd3a305777b5dc8262c675bf23c8dc5acfe8c4fa1ca8acdd956cdcbdf1fb2' +
+  'f879b37dc0ed8ec51815ff02b98eefde44ac79f886902ea69e5ed2561e9e' +
+  'eb2a74ec1de677b285f8108f25e9f34cc826fbbd1ad091d231dc73eaf28a' +
+  'e02f09ad84ec9a38d8a7e28bea26fb7d4db20eecd075b5b261ca7320af94' +
+  'cd58ba4b26d895df4fd6f68bd4d82acdcb35557012e2f69739ce8cf4a66e' +
+  'bf4550ee50f6c9cec642d66fef71495a';
+
+export const rsaSignVerifyObjects = {
```

Comment:
If these are from nodejs tests, can you add link to the test file as a comment?

Reply:
The fixtures are from node.js' tests but the actual test I rolled myself. I haven't started porting Node.js' actual tests yet.